plugins/typo_squatter/komand_typo_squatter/util/utils.py

```diff
@@ -1,14 +1,49 @@
+from insightconnect_plugin_runtime.exceptions import PluginException
+import subprocess  # nosec
+import validators
+import json
 import re
-import tqdm
-import entropy
+import math
 from tld import get_tld
 from Levenshtein import distance
-from termcolor import colored, cprint
-
 from .suspicious import keywords, tlds
 
 
-def score_domain(domain):
+def check_for_squatters(domain: str, flag: str, logger) -> list:
```

Comment:
why is this code in the util file rather than in the action file?

Reply:
I wanted to minimize the amount of code in the action.

Reply:
Right, but the code in this util file IS the action. One reason to keep this separate is because for simplicities sake- future developers will naturally want to fix the code in the action file if the action is broken. Another reason is if another developer wants to fix another action in this plugin at the same time, there will be merge conflicts when trying to merge the branches together due to both working on the same file.

Reply:
If you see this as something that will be re-usable code, and not an action, then we can keep it in the util.py file

Reply:
@pmara-r7 You're right, then I'll move this method back to the action file.